Libraries/Components/ScrollResponder.js

```diff
@@ -112,6 +113,15 @@ type State = {
 };
 type Event = Object;
 
+function isTagInstanceOfTextInput(tag) {
+  var instance = getInstanceFromNode(tag);
+  return instance && (
+    instance.constructor.displayName === 'AndroidTextInput' ||
```

Comment:
The issue is that we have the internal instance and not the current element, it would be possible to get it by using a private field but I preferred not to. Is there a proper way to get the element from the viewTag?

Looks like we could also use uiViewClassName, which seems better than displayName. Here what the instance looks like.

![image](https://cloud.githubusercontent.com/assets/2677334/20566690/f9dcee7e-b163-11e6-9162-28ea37b53827.png)